lib/signaling/v2/peerconnection.js

```diff
@@ -10,7 +10,8 @@ const {
   getStats: getStatistics
 } = require('@twilio/webrtc');
 
-const { guessBrowser } = require('@twilio/webrtc/lib/util');
+const util = require('@twilio/webrtc/lib/util');
+const { guessBrowser } = util;
```

Comment:
Any reason why this change was made. The previous require call was just one line.

Reply:
This change was made to enable unit testing against different browsers. By importing `util` separately  we can  now mock guessBrowser [here](https://github.com/twilio/twilio-video.js/blob/8c575659750af12476891dc36071fae1a08c081a/test/unit/spec/signaling/v2/peerconnection.js#L87)